activerecord/lib/active_record/encryption/encryptor.rb

```diff
@@ -69,7 +69,7 @@ def encrypted?(text)
       private
         DECRYPT_ERRORS = [OpenSSL::Cipher::CipherError, Errors::EncryptedContentIntegrity, Errors::Decryption]
         ENCODING_ERRORS = [EncodingError, Errors::Encoding]
-        THRESHOLD_TO_JUSTIFY_COMPRESSION = 140.bytes
+        THRESHOLD_TO_JUSTIFY_COMPRESSION = 140
```

Comment:
140 what? (This is the reason `#bytes` exists and it’s used here.)

Reply:
I agree: the code is no longer self-documenting.

Reply:
There is something better than self-documenting code. Actually documenting the code! ;)

Reply:
I have added a single-line comment explaining that the constant is in bytes and what it does.
7605def0511df64cafb5defc76ff292fb25b79c3